website/docs/core-concepts/writing-code/workflows.md

```diff
@@ -2,155 +2,161 @@
 
-```
- {{getAllUsers.data}}
+```javascript
+showAlert("Data saved successfully.");
 ```
 
-:::info
-The [table data property of a table](../../reference/widgets/table/#table-data) requires a **JSON Array** so verify that your API returns a **JSON Array**.
-:::
+<VideoEmbed host="youtube" videoId="tjJIDkoCyQE" title="Global Functions" caption="Global Functions"/> 
 
-A `{{API.run()}}` method, if supplied to the [Table data](../../reference/widgets/table/#table-data), will throw an error, as table data is a Sync field and cannot perform the execution. However, you can read the response generated by the API by accessing the data property `{{API.data}}` of an API. When you bound an API data property to a widget, Appsmith executes the API **on page load**. You can modify the API settings if you wish not to execute the API **on page load**.
+You can only use the **global actions** provided out-of-the-box by Appsmith in the **async** **fields**.  
 
-**Using a JSObject**
+##### Handle success or error
 
-You have a JSObject function that fetches all Users' information, filters data for `firstname` & `email,` and then returns the result so you can bind it to the table widget and display information.
+When configuring API and query executions, it's important to provide feedback to the user about the outcome of the action. This can be done by displaying success messages when an action is completed successfully or error messages when there are issues with the execution.
 
- <VideoEmbed host="youtube" videoId="8mVQS6uaR6M" title="Using a JSobject" caption="Using a JSobject"/>  
+<VideoEmbed host="youtube" videoId="4aEMFU1r1yg" title="Setup Success or Error messages" caption="Setup Success or Error messages"/>
 
+You can configure the action to be performed once a query returns with a success or an error. The success or error returned by the API or query can be determined by the HTTP status code or the query response status. These status code are bound to `onSuccess` and `onError` attributes of events and available in property pane to be setup for showing informative messages to end users.
 
-To read the data by calling a JSObject function: Create a JSObject - `getFilteredUsersList` on Appsmith by adding a `JSObject` from `Explorer` >> Click `(+)` Query/JS >> Select `New JS Object` >> Rename to `getFilteredUsersList` (or select an existing JSObject) You can add your API or Query Call to the function call and filter the data for the required details as below:
+You can also choose to write custom JavaScript code by enabling the `JS` label next to event, and add the code you wish to trigger on success or when an error is generated, as shown below:
```

Comment:
Should we be talking about the JS label here?  I see that we are introducing it in complex workflows in the section below

Reply:
As discussed, we are keeping this as is.